### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by AndriiYelieva**
[DEMO LINK](https://AndriiYelieva.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const getPartnerLabel = () => {
```
Comment (Anastasiia-Svintsova) [inline src/components/Person/Person.jsx:6]: You don't really need a function for it. Jus create a constant - `partnerLabel` for example.

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,31 @@
...
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {getPartnerLabel()}
```
Comment (Anastasiia-Svintsova) [inline src/components/Person/Person.jsx:27]: It's generally recommended to compute the values outside of JSX and store them in variables or state before rendering JSX.

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- Deploy your changes 
image

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Well done

**Pull request closed without merging**